tests/js/jest.setup.js

```diff
@@ -21,7 +21,6 @@
 // See https://github.com/testing-library/jest-dom.
 import 'jest-extended';
 import '@testing-library/jest-dom';
-import 'jest-extended';
```

Comment:
This fixes eslint warning. Should be good with https://xwp.slack.com/archives/CCCAUAH6F/p1588745737296900